python/tk_multi_publish2/dialog.py

```diff
@@ -235,8 +246,31 @@ def __init__(self, parent=None):
         self._display_action_name = self._bundle.get_setting("display_action_name")
         self.ui.publish.setText(self._display_action_name)
 
+        # UI tweaks based on the 'manual_load_enabled' property
+        # disabling drag & drop and browsing for files or folders.
+        #
+        # hide the drop area ( dashed-line border + plus icon )
+        self.ui.large_drop_area.setVisible(self.manual_load_enabled)
+        self.ui.large_drop_area_label.setVisible(self.manual_load_enabled)
+        self.ui.label.setVisible(self.manual_load_enabled)
+        self.ui.drop_area_browse_file.setVisible(self.manual_load_enabled)
+        self.ui.drop_area_browse_seq.setVisible(self.manual_load_enabled)
+        self.ui.browse.setVisible(self.manual_load_enabled)
```

Comment:
this is on the main page, i'd move it down and separate it from the drag/drop page.

Reply:
Done